scripts/test-acceptance.sh

```diff
@@ -53,4 +53,4 @@ if [[ "${test_name}" = "accessibility" ]]; then
 fi
 
 echo "Running acceptance tests from ${test_name}.py against the sandbox..."
-nosetests ../acceptance/${test_name}.py --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
+nosetests ../acceptance/${test_name}.py:MultipleOpenAssessmentTest --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
```

Comment:
@mushtaqak This is a good idea for debugging your new tests on jenkins quickly, but please be sure to have a full run of all tests (the changes to this file backed out) before merging.

The `No such test MultipleOpenAssessmentTest` failure you're seeing is happening because jenkins runs acceptance and accessibility tests, and there is no `MultipleOpenAssessmentTest` for accessibility.